### DISCUSSION TITLE
MDN community call time: earlier or later?

### DISCUSSION THREAD

**Discussion by pepelsbey** [mdn/mdn-community#678] [Announcements]
[The next community call](https://github.com/mdn/community-meetings/blob/main/2024-04-29/agenda.md) is planned for April 29th at the usual time, 17:00 CEST (UTC+2). While the date and time for this one is set, we keep thinking about making it easier for everyone to attend. So, would you attend the MDN community call if it started earlier or later? We know that it’s almost impossible to find a perfect spot convenient for everyone, but maybe a small adjustment could make it easier. Let us know!

Feel free to comment with your exact suggestion or any ideas you have for making the timing work for as many people as possible.

**Comment by chrisdavidmills** [comment]
The community calls are at a perfect time for me! ;-)

**Comment by hochan222** [comment]
Perfect time for me too. But if it's late, it's a little difficult.
(Today's meeting time in South Korea is 12 a.m. )
